<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaspCamMW</class>
 <widget class="QMainWindow" name="RaspCamMW">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Programacion Remota Kit Nexys 2 Digilent</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="videow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>640</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_address">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Servidor:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>20</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Video</string>
     </attribute>
     <widget class="QGroupBox" name="gp_Controls">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>70</y>
        <width>220</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Controles</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>199</width>
         <height>95</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="1">
         <widget class="QPushButton" name="btnSnap">
          <property name="text">
           <string>Capturar</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pics/snapshot.png</normaloff>pics/snapshot.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnStop">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pics/stop.png</normaloff>pics/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="btnRecord">
          <property name="text">
           <string>REC</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pics/record.png</normaloff>pics/record.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>pics/play.png</normaloff>
            <normalon>pics/play.png</normalon>
            <disabledoff>pics/play.png</disabledoff>
            <disabledon>pics/play.png</disabledon>
            <activeoff>pics/play.png</activeoff>
            <activeon>pics/play.png</activeon>
            <selectedoff>pics/play.png</selectedoff>
            <selectedon>pics/play.png</selectedon>pics/play.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnScan">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>191</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Video streaming</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblprocess">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>300</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Kit Nexys2</string>
     </attribute>
     <widget class="QPushButton" name="btnConect">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Conectar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProgram">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>190</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Programar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBitFile">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>80</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>.bit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>921</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
